src/mesh/vtk_helper.jl

```diff
@@ -0,0 +1,108 @@
+
+"""
+    n_verts_between(n, from, to, dim)
+
+Compute the coordinates of n equally distributed points between the points
+given by `from` and `to`. `dim` is the dimension of `from` and `to`. 
+Inspired by: https://github.com/ju-kreber/paraview-scripts/blob/master/node_ordering.py
+"""
+function n_verts_between(n, from, to)
+    if n <= 0
+        return
+    end
+    dim = length(from)
+    @assert length(from) == length(to)
+    edge_verts = [LinRange(from[i], to[i], n+2)[2: 1+n] for i in 1:dim]
+    return edge_verts
+end
+
+
+"""
+    triangle_vtk_order(corner_verts, order, dim, skip = false)
+
+Compute the coordinates of a `VTK_LAGRANGE_TRIANGLE` of a triangle or order `order` 
+defined by the coordinates of the vertices given in `corner_verts`. `dim` is the
+dimension of the coordinates given. If `skip` is set to true, the coordinates
+of the vertex- and edge-points aren't computed, which can be used to compute
+points of a `VTK_LAGRANGE_WEDGE`
+Inspired by: https://github.com/ju-kreber/paraview-scripts/blob/master/node_ordering.py
+"""
+function triangle_vtk_order(corner_verts, order, dim, skip = false)
+    if order < 0
+        return nothing
+    end
+    if order == 0
+        # For recursion
+        return corner_verts[:,1]
+    end
+    #Corner vertices
+    coords = Matrix{Float64}(undef, 0, dim)
+    if skip == false
+        coords = corner_verts
+    end
+    #edge vertices
+    num_verts_on_edge = order -1 
+    # edges in vtk-order
+    edges = [(1,2), (2,3), (3,1)]
```

Comment:
```suggestion
    edges = rd.fv
```
I think you can get edges from `rd.fv` (`fv` = face vertices). Can you check if this works? If so, it would be good to match `rd.fv` in case we change the order.